configure

```diff
@@ -527,8 +527,11 @@ def configure_node(o):
   o['variables']['clang'] = 1 if is_clang else 0
 
   if not is_clang and cc_version != 0:
-    o['variables']['gcc_version'] = 10 * cc_version[0] + cc_version[1]
-
+    try:
+        o['variables']['gcc_version'] = 10 * cc_version[0] + cc_version[1]
+    except IndexError:
```

Comment:
``o['variables']['gcc_version'] = 10 * cc_version[0]``
This line always run, because ``cc_version[0]`` refers the major version of gcc.
Now I think, we only need to check if ``cc_version[1]`` exists. If exists, use your value, if not exists show a warning message. Like the code above.
I've used ``try .. except`` because Python encourages use of  ``try``. 
I've tested ``if  .. else`` way and my ``try ... except`` way, both ways have solved this issue.